src/state/queries/profile-feedgens.ts

```diff
@@ -38,5 +40,21 @@ export function useProfileFeedgensQuery(
     initialPageParam: undefined,
     getNextPageParam: lastPage => lastPage.cursor,
     enabled,
+    select(data) {
+      return {
+        ...data,
+        pages: data.pages.map(page => {
```

Comment:
Can you check if this is going to lead to re-renders for every pagination? See the hack we added to `select` in `post-feed` to avoid that there. I guess it's not super important here per se but would be good to verify.

Reply:
Did some digging on this and even did the approach from `post-feed`, but then realized it was really just `renderItem` on the `List` not being memoized 🙃

Reply:
But yeah you were right, there were re-renders happening here, though not bc of the query 😆